src/simplewallet/simplewallet.cpp

```diff
@@ -6538,14 +6554,46 @@ bool simple_wallet::update_lns_mapping(const std::vector<std::string>& args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
-bool simple_wallet::print_lns_name_to_owners(const std::vector<std::string>& args)
+bool simple_wallet::lns_make_update_mapping_signature(const std::vector<std::string> &args)
+{
+  if (!try_connect_to_daemon())
+    return true;
+
+  if (args.size() < 2)
+  {
+    PRINT_USAGE(USAGE_LNS_MAKE_UPDATE_MAPPING_SIGNATURE);
+    return true;
+  }
+
+  std::string const &new_value = args[args.size() - 1];
+  std::string name;
+  if (!parse_lns_name_string(args, 0, (args.size() - 2), name))
+  {
+    PRINT_USAGE(USAGE_LNS_MAKE_UPDATE_MAPPING_SIGNATURE);
+    fail_msg_writer() << "lns name didn't start or end with quotation marks (\")";
+    return true;
+  }
+
+  SCOPED_WALLET_UNLOCK();
+  crypto::generic_signature signature;
+  std::string reason;
+  if (m_wallet->lns_make_update_mapping_signature(lns::mapping_type::session, name, new_value, signature, &reason))
+    tools::success_msg_writer() << "signature=" << epee::string_tools::pod_to_hex(signature);
+  else
+    fail_msg_writer() << reason;
+
+  return true;
+}
+//----------------------------------------------------------------------------------------------------
+static char constexpr NULL_KEY_STR[] = "0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
This would be more user friendly if we just printed "(none)" instead of a bunch of 0s.